.travis.yml

```diff
@@ -21,7 +21,7 @@ before_deploy:
 - ./.travis/install_install4j
 - ENGINE_VERSION=$(grep "engine_version" game_engine.properties | sed 's/.*= *//g')
 - TAGGED_VERSION="$ENGINE_VERSION.$TRAVIS_BUILD_NUMBER"
-- ./.travis/build_installers $TAGGED_VERSION
+- TAGGED_VERSION=$TAGGED_VERSION ./gradlew release
```

Comment:
I changed the build script to use a Gradle project property instead of an environment variable to define the version.  However, while working through this exercise, a few other issues came up.  They are minor and easily resolved, but it may require a team discussion to decide the best path forward.

Therefore, I'm going to propose that the change to a Gradle project property be postponed to a separate PR.